pom.xml

```diff
@@ -1,27 +1,104 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
-   <modelVersion>4.0.0</modelVersion>
-   <groupId>car-app</groupId>
-   <artifactId>car-app</artifactId>
-   <version>0.0.1-SNAPSHOT</version>
-   <name>car-app</name>
-   <description>car-app</description>
-   <dependencies>
-      <!-- https://mvnrepository.com/artifact/org.hibernate/hibernate-core -->
-      <dependency>
-         <groupId>org.hibernate</groupId>
-         <artifactId>hibernate-core</artifactId>
-         <version>5.4.27.Final</version>
-      </dependency>
-      <dependency>
-         <groupId>org.slf4j</groupId>
-         <artifactId>slf4j-log4j12</artifactId>
-         <version>1.7.2</version>
-      </dependency>
-      <dependency>
-         <groupId>org.postgresql</groupId>
-         <artifactId>postgresql</artifactId>
-         <version>42.2.18</version>
-      </dependency>
-   </dependencies>
-</project>
\ No newline at end of file
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>car-app</groupId>
+  <artifactId>car-app</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>car-app</name>
+  <packaging>war</packaging>
+  <description>car-app</description>
+
+  <dependencies>
+
+    <dependency>
+       <groupId>org.hibernate</groupId>
+       <artifactId>hibernate-core</artifactId>
+       <version>5.4.27.Final</version>
+    </dependency>
+
+		<dependency>
+		    <groupId>org.hibernate</groupId>
+		    <artifactId>hibernate-core</artifactId>
+		    <version>4.1.5.Final</version>
+		</dependency>
+
+  		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-log4j12</artifactId>
+			<version>1.7.2</version>
+		</dependency>
+
+		<dependency>
+		    <groupId>org.postgresql</groupId>
+		    <artifactId>postgresql</artifactId>
+		    <version>42.2.18</version>
+		</dependency>
+
+		<dependency>
+            <groupId>com.sun.jersey</groupId>
+            <artifactId>jersey-servlet</artifactId>
```

Comment:
As mentioned in the daily, please switch to javaee dependency or even better jakartaee dependency.

https://mvnrepository.com/artifact/javax/javaee-api/8.0.1
https://mvnrepository.com/artifact/jakarta.platform/jakarta.jakartaee-api/8.0.0